pkg/cmd/ssh_azure.go

```diff
@@ -76,8 +75,12 @@ func sshToAZNode(nodeName, path, user, pathSSKeypair string, sshPublicKey []byte
 	fmt.Println("")
 
 	key := filepath.Join(pathSSKeypair, "key")
-	sshCmd := fmt.Sprintf("ssh -i " + key + " -o StrictHostKeyChecking=no " + node)
-	cmd := exec.Command("bash", "-c", sshCmd)
+	args := []string{"-i" + key, "-oStrictHostKeyChecking=no ", node}
```

Comment:
```suggestion
	args := []string{"-i" + key, "-oStrictHostKeyChecking=no", node}
```
emtpy space after `-oStrictHostKeyChecking=no` can be removed